src/debug/seer-integration.js

```diff
@@ -75,6 +75,14 @@ export const layerEditListener = cb => {
   });
 };
 
+export const removeLayer = id => {
+  if (!window.__SEER_INITIALIZED__ || !id) {
```

Comment:
Nit: I personally prefer to avoid returns when it doesn't complicate the code too much.
```
if (isSeerInitialized() && id) {
  seer.delete...
}
```
Would look tighter to me here.